packages/@lwc/shared/src/signals.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+export const SIGNAL_SYMBOL = Symbol.for('experimental-signal');
```

Comment:
we can bike-shed on the name and make it scarier I suppose!

Reply:
`experimental-signal` is fine IMO

Reply:
`ooky-spooky-scary-experimental-signal`